ts/WoltLabSuite/Core/Ajax/Backend.ts

```diff
@@ -135,14 +143,19 @@ class BackendRequest {
       init.method = "POST";
 
       if (this.#payload) {
-        if (this.#payload instanceof FormData) {
+        if (this.#payload instanceof Blob) {
+          init.headers!["Content-Type"] = "application/octet-stream";
```

Comment:
Is the `!` actually necessary here? `init.headers` appears to always be initialized.

Reply:
It is defined as `HeadersInit | undefined` 🤷